frontend/src/app/components/action-plan/action-plan.component.ts

```diff
@@ -22,12 +22,13 @@ export class ActionPlanComponent {
   listItems!: QueryList<ElementRef>;
 
   constructor(private actionService: ActionService,
-    public dialogService: DialogService) {}
+    public dialogService: DialogService) {
+  }
 
   handleKeyDown(event: Event, currentIndex: number) {
     let newIndex = currentIndex;
     if ((event as KeyboardEvent).key === 'ArrowDown') {
-      if (newIndex + 1 <= this.control.getValue()!.length - 1) {
+      if (newIndex + 1 <= (this.control.getValue() ?? []).length - 1) {
```

Comment:
this does not have the same behavior.... we want to assert not null and have the programm throw an error if it is null. The new implementation provides a fallback and we do not catch the error, which might lead to confusing behavior.